src/Datadog.Trace.ClrProfiler.Managed/Integrations/MongoDbIntegration.cs

```diff
@@ -122,7 +122,7 @@ public static object ExecuteAsync(object wireProtocol, object connection, object
             TargetMethod = nameof(ExecuteAsync),
             TargetAssembly = MongoDbClientAssembly,
             TargetType = IWireProtocolGeneric,
-            TargetSignatureTypes = new[] { ClrNames.Ignore, ClrNames.Ignore, ClrNames.Ignore },
+            TargetSignatureTypes = new[] { "System.Threading.Tasks.Task`1<T>", "MongoDB.Driver.Core.Connections.IConnection", ClrNames.CancellationToken },
```

Comment:
You might consider adding `System.Threading.Tasks.Task`1<T>` to ClrNames as well, since that is the other Task type. There are no other open-generic Task types.

Reply:
I did think about that and decided to defer the decision until later until we figure out a final convention for what we want to do with generic arg names, if that's okay.

Reply:
Yup that sounds good!